api/docgen/exampledata/extendedHeader.json

```diff
@@ -46,7 +46,7 @@
     }
   },
   "commit": {
-    "height": 67374,
+    "height": "67374",
```

Comment:
why is this change necessary here btw?

Reply:
it just happened

Reply:
it's from some other commit that I don't know why it happened :D I'll revert